src/NftGallery.tsx

```diff
@@ -225,11 +278,31 @@ export const NftGallery: React.FC<NftGalleryProps> = ({
   // Handles fetching of assets via OpenSea API.
   useEffect(() => {
     if (showcaseMode) {
-      loadShowcaseAssets(ownerAddress, openseaApiKey);
+      loadShowcaseAssets(
+        ownerAddress,
+        openseaApiKey,
+        isProxyApi,
+        apiUrl,
+        autoRetry
+      );
     } else {
-      loadAssets(ownerAddress, openseaApiKey, currentOffset);
+      loadAssets(
+        ownerAddress,
+        openseaApiKey,
+        isProxyApi,
+        apiUrl,
+        autoRetry,
+        currentCursor
+      );
     }
-  }, [showcaseMode, ownerAddress, openseaApiKey, currentOffset]);
+  }, [
+    showcaseMode,
+    ownerAddress,
+    openseaApiKey,
+    isProxyApi,
+    apiUrl,
```

Comment:
Missing `autoRetry` as a hook dependency:

```suggestion
    apiUrl,
    autoRetry,
```